interpreter/cling/include/cling/Interpreter/Value.h

```diff
@@ -129,21 +188,18 @@ namespace cling {
         EStorageType storageType = V.getStorageType();
         switch (storageType) {
```

Comment:
What's the point of this, now that there are many, many more storage types?

Reply:
Yep, that will go away in another commit if this goes in.

Reply:
github mark the code as "outdated".   Did the resolution of this go into this PR or not?  If it did not, should we open an issue to keep track of it?

Reply:
Vassil reports that this code is already gone as part of this PR.